src/pages/index.js

```diff
@@ -219,3 +220,44 @@ addProfilePopup.setEventListeners();
 const nameSelector = ".profile__title";
 const aboutMeSelector = ".profile__description";
 const userInfo = new UserInfo({ nameSelector, aboutMeSelector });
+
+//API 
+const api = new Api({
+  baseURL: "https://around.nomoreparties.co/v1/group-1",
+  authToken: "eeb6862d-8337-45ca-b804-a54d677deb3a"
+})
+
+api.removeCard("cardID:any").then(res => console.log(res));
+
+api.getCardList().then(res => console.log(res));
```

Comment:
Needs correction: please remove this extra api call

Reply:
Hi "Outdated" does this mean that this error is fixed ?